### ISSUE TITLE
Are org.apache.commons libraries considered third-party?

### ISSUE THREAD

**Issue by WilliamHaiweiGu** [nus-cs2103-AY2223S2/forum#368]
[Apache Commons](https://commons.apache.org) has many Java libraries. Considering its formalness and wide usage, I never thought about this but are Apache Commons libraries considered third-party?

If so, can I add it to the acknowledgment section of my TP docs right now? Thank you!

**Comment by damithc** [comment]
@WilliamHaiweiGu Yes, they are. OK, you can add the acknowledgement in the code, but keep the previous DG submission. Otherwise you'll incur a late submission penalty.
